src/runtime_src/core/tools/common/XBUtilities.h

```diff
@@ -32,83 +32,22 @@
 #include <boost/algorithm/string.hpp>

```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.